.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
[Optional]
- [ ] `.eslintrc`, `.stylelintrc.json` files only suggested to use for configuring for your local linters. You should not commit them and you should include them in your `.gitignore` file